wagtail/admin/tests/pages/test_explorer_view.py

```diff
@@ -582,6 +582,15 @@ def test_filter_by_page_type(self):
             f"content_type={page_type_pk}",
         )
 
+        # "Page" should not be listed as a content type
+        soup = self.get_soup(response.content)
+        page_type_labels = {
+            list(label.children)[-1].strip()
```

Comment:
The fact that `BeautifulSoup` ignores text within `<template>` tags has tripped me a few times when writing tests for such elements. I've found a way to change the behaviour via `get_soup()`, will see about making a PR for it...

Reply:
Aha, thanks for the explanation! I didn't realise the `<template>` was the cause, and was wondering why BeautifulSoup was being unusually un-cooperative with my attempts to write this in a more direct way :-)

Reply:
Yep, if you're interested: https://www.crummy.com/software/BeautifulSoup/bs4/doc/#get-text

> As of Beautiful Soup version 4.9.0, when lxml or html.parser are in use, the contents of <script>, <style>, and <template> tags are generally not considered to be ‘text’, since those tags are not part of the human-visible content of the page.